server/src/main/java/com/lunchtime/service/impl/PersonServiceImpl.java

```diff
@@ -12,16 +13,22 @@
 import org.springframework.security.crypto.bcrypt.BCrypt;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
 
 import javax.persistence.NonUniqueResultException;
 import java.util.Optional;
+import java.util.UUID;
 
 @Service
 @RequiredArgsConstructor
 public class PersonServiceImpl implements PersonService {
     private final PersonRepository personRepository;
+    private final RoleRepository roleRepository;
     private final PersonMapper personMapper;
     private final BCryptPasswordEncoder bcryptPasswordEncoder;
+    private final MailSender mailSender;
+
+    String webPage = "http://35.234.68.126";
```

Comment:
Don't hardcode staging url in source code. Use different application.properties for different environments

Reply:
This doesn't work with `cors.urls` even if I rename variable in `application-dev.properties` from `cors.urls` to `cors.urls.dev`.
```
    @Value("${cors.urls.dev}")
    String webPage;
```
Mey be you know how to resolve this?

Reply:
Check file `WebConfiguration.java`. In dev environment it will be `http://localhost:3000` in prod - `staging url`